test/cases/testdata/time/test-time-0972.yaml

```diff
@@ -0,0 +1,14 @@
+cases:
+- data:
+  modules:
+  - |
+    package generated
+
+    p = __local0__ {
+      time.diff(time.parse_ns("2006-01-02", "2020-02-02"), time.parse_ns("2006-01-02", "2020-03-01"), __local1__)
```

Comment:
Perhaps you could convert "2006-01-02" to a variable called `layout` to make the tests more readable?